features/recommendations.feature

```diff
@@ -129,11 +138,31 @@ Feature: The recommendation service back-end
         And I press the "Retrieve" button
         Then I should see the message "404 Not Found: Recommendation with id '123456' was not found."
 
-    Scenario: List all Recommendations
+    Scenario: Update a Reccommendation
         When I visit the "Home Page"
-        And I press the "List" button
+        And I press the "Clear" button
+        And I set the "Product A SKU" to "GQGEsdfq"
+        And I press the "Search" button
         Then I should see the message "Success"
-        And I should see "HYJtLnYf" in the results
-        And I should see "GQGEsdfq" in the results
-        And I should see "FQEFQrQs" in the results
-        And I should see "dasdfeaQ" in the results
\ No newline at end of file
+        And I should see "GQGEsdfq" in the "Product A SKU" field
+        And I should see "cUafQfef" in the "Product B SKU" field
+        And I should see "CROSS_SELL" in the "Recommendation type" field
+        When I change "Product A SKU" to "FWiNenfo"
+        And I press the "Update" button
+        Then I should see the message "Success"
+        When I copy the "Id" field
+        And I press the "Clear" button
+        And I paste the "Id" field
+        And I press the "Retrieve" button
+        Then I should see the message "Success"
+        And I should see "FWiNenfo" in the "Product A SKU" field
+        When I press the "Clear" button
+        And I press the "Search" button
```

Comment:
If you are doing an empty search, shouldn't it be the “List” button?

Reply:
The functionality of List and empty Search is totally the same in our project.